test/libyul/yulOptimizerTests/fullSuite/clear_after_if_continue.yul

```diff
@@ -12,7 +12,7 @@
 // {
 //     {
 //         let y := mload(0x20)
-//         for { } iszero(iszero(and(y, 8))) { if y { revert(0, 0) } }
+//         for { } and(y, 8) { if y { revert(0, 0) } }
```

Comment:
How come?

Reply:
Broken on ``develop_060`` in general... we might want to fix it in a separate PR... it's a separate commit here at least.

Reply:
ok